packages/next-swc/Cargo.lock

```diff
@@ -59,9 +59,9 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.51"
+version = "1.0.52"
```

Comment:
Also, this PR seems to include updates of dependencies, but it should be done with own PR.
Can you revert it?